### ISSUE TITLE
LayoutLMv3 Tokenizer and umlauts

### ISSUE THREAD

**Issue by MoPl90** [NielsRogge/Transformers-Tutorials#155]
I've noticed strange behaviour with the LayoutLMv3 tokenizer. It seems to split umlauts like ü into two tokens and assigns both of them a non-negative label. This makes it very difficult to reconstruct word-level predictions in a straight-forward manner (which rely on the pad_token_label=-100). Here's an example:

```
from transformers import AutoTokenizer

tokenizer = AutoTokenizer.from_pretrained("microsoft/layoutlmv3-base", apply_ocr=False)
tokenizer(["übereinkunft"], boxes=[[0,10,20,30]], word_labels=[0], padding=False, truncation=False)
```
yields
```
{"input_ids": [0, 952, 4394, 428, 2816, 4291, 879, 2543, 2], "attention_mask": [1, 1, 1, 1, 1, 1, 1, 1, 1], "bbox": [[0, 0, 0, 0], [0, 10, 20, 30], [0, 10, 20, 30], [0, 10, 20, 30], [0, 10, 20, 30], [0, 10, 20, 30], [0, 10, 20, 30], [0, 10, 20, 30], [0, 0, 0, 0]], "labels": [-100, 0, 0, -100, -100, -100, -100, -100, -100]}
```
Note the two labels of "0". Is this intentional or a bug? In the former case, what is the intended way to obtain word-level predictions?

**Comment by NielsRogge** [comment]
Hi, `microsoft/layoutlmv3-base` was trained on English-only data. I'm not sure it will work well on German words.

**Comment by Kuz-man** [comment]
Maybe you can try the `microsoft/layoutxlm-base` tokenizer, it is trained on German as well.
